feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/ui/PreviewScreenComponents.kt

```diff
@@ -430,13 +430,24 @@ fun ZoomScaleText(zoomScale: Float) {
     )
     Text(
         modifier = Modifier
+            .padding(horizontal = 10.dp)
             .alpha(contentAlpha.value)
             .testTag(ZOOM_RATIO_TAG),
         text = "%.1fx".format(zoomScale),
         fontSize = 20.sp
```

Comment:
Since we'll be setting a lot of font sizes for all of this debug text, you can wrap the entire block of all text like:
```
CompositionLocalProvider(
    LocalTextStyle provides LocalTextStyle.current.copy(fontSize = 20.sp)
) {
    if (showZoomLevel) {
        ZoomScaleText(zoomLevel)
    }
    if (previewUiState.isDebugMode) {
        CurrentCameraIdText(physicalCameraId, logicalCameraId)
    }
}
```
Then you will no longer need to set it on each element.